Services/Music_Portal.Services.Services/ArtistService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Domain.Core;
+using Infrastructure.Data;
+using Services.Interfaces;
+
+namespace Services.Services
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Services.Services`